### PR TITLE
login.js: Redirect back to page that required Auth

### LINKED ISSUE #370: Route does not forward after login
If you open a link directly so a room i.e. https://example.com/#!/room/5610f3a75438d2cb0c80b8aa, after login you are taken to the main page not the room. The route should forward and after login be taken directly to the specified room.

### PR DESCRIPTION
**Pull request by Shelnutt2**
If a hash exists on the location, the include it in the redirect after login (fixes #370)

### CODE REVIEW

File: media/js/login.js [hunk 1]
```diff
@@ -25,7 +25,13 @@
                     });
                     if ($form.data('refresh')) {
                         setTimeout(function() {
-                            window.location = './';
+                            if(window.location.hash != undefined){
```
Comment (hhaidar) [inline media/js/login.js]: I don't think location.hash will ever be undefined (empty string in chrome). I could be wrong.

File: media/js/login.js [hunk 2]
```diff
@@ -25,7 +25,13 @@
...
+                                var hash = window.location.hash;
+                                window.location = './'+hash;
```
Comment (hhaidar) [inline media/js/login.js]: Can you change this section `to './' + (hash || '')`

**Comment by hhaidar** [comment]
@Shelnutt2 just some minor stuff and we can get this in

**Comment by sibartlett** [comment]
Merged

**Pull request closed without merging**